components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
+	padding: 1.5em;
+
+
+	&__title {	
+	font-size: 2em;
```

Comment:
Although ems work fine for setting these font sizes, you could consider using rems as well. Basically the same as ems but is dependent on the document's base font size rather than its parent's. To implement it, you can use the function calc-toRem(n) where "n" is the size in pixels. So you could do:
```scss
&__title{
    font-size:calc-toRem(32);
}
```